package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-standard-react",
   "description": "JavaScript Standard Style React/JSX support - ESLint Shareable Config",
-  "version": "9.2.0",
+  "version": "10.0.0",
```

Comment:
Version bump is usually done by the maintainer on the release commit so you may want to revert this.

Reply:
Please revert this change.